Update wifi_hal_ap.h

Athira-Vij opened the pull request:
RDK-Hackathon2025: Easy Connect application
Reason for change: Make DPP related changes for vap structure in halinterface. Test Procedure: Build should be success.
Risks: None
Signed-off-by:athira.vijayan@thinkpalm.com

Athira-Vij committed:
Update wifi_hal_ap.h

RDK-Hackathon2025: Easy Connect application
Reason for change: Make DPP related changes for vap structure in halinterface.
Test Procedure: Build should be success.
Risks: None
Signed-off-by:athira.vijayan@thinkpalm.com

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/rdkcentral/rdkb-halif-wifi?pullRequest=27) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/rdkcentral/rdkb-halif-wifi?pullRequest=27) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/rdkcentral/rdkb-halif-wifi?pullRequest=27) it.</sub>

Athira-Vij committed:
Update wifi_hal_ap.h

RDK-Hackathon2025: Easy Connect application
Reason for change: Make DPP related changes for vap structure in halinterface.
Test Procedure: Build should be success.
Risks: None
Signed-off-by:athira.vijayan@thinkpalm.com

Athira-Vij commented on include/wifi_hal_ap.h:
```diff
@@ -3059,6 +3091,7 @@ typedef struct
     CHAR bridge_name[WIFI_BRIDGE_NAME_LEN]; /**< Bridge name. */
     wifi_vap_mode_t vap_mode;      /**< VAP mode. */
     wifi_vap_name_t repurposed_vap_name; /**< Repurposed VAP name. */
+    wifi_vap_dpp_t      vap_dpp;  /**< DPP information*/
```
this can co exist with wifi_front_haul_bss_t. If moving I think it could be moved to a structure within wifi_front_haul_bss_t
